build/resources/main/liquibase/1.0/changelog-1.0.1.yml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+  - changeSet:
+      id: Create shedlock table
+      author: Taleh Bakhshizada
+      changes:
+        - createTable:
+            tableName: shedlock
+            columns:
+              - column:
+                  name: name
+                  type: varchar(64)
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: shedlock_pk
+                    nullable: false
+              - column:
+                  name: lock_until
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: locked_at
+                  type: timestamp
+              - column:
+                  name: locked_by
+                  type: varchar(255)
+              - createIndex:
```

Comment:
I think you don't index here, because postgres have a default index based on primaryKey or unique fields